reference-implementation/lib/writable-stream.js

```diff
@@ -505,16 +505,6 @@ class WritableStreamDefaultWriter {
       return Promise.reject(defaultWriterBrandCheckException('write'));
     }
 
-    const stream = this._ownerWritableStream;
```

Comment:
I am sad about moving these checks as I like the separation of invariant-checking in the public API only. It seems like at least the first check, of stream being undefined, can stay here. What about the second one?